client-sdk/ts-web/jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    testMatch: ['**/test/**/*.ts'],
+    testEnvironment: 'node',
+    transform: {
+        '^.+\\.[jt]s?$': 'babel-jest',
```

Comment:
Do you have any reason to prefer this over [ts-jest](https://www.npmjs.com/package/ts-jest)?

Reply:
It's faster, and avoids type-checking twice as well (see the PR's body)

Reply:
Thanks for the writeup. Are you sure you don't want typechecked tests? I find it beneficial to have early feedback that I'm using the APIs incorrectly [when writing tests].

Reply:
You're right - In my other projects we usually have a separate step with a different tsconfig to typecheck the tests as well but I think this would be a bit overkill right now - I'll amend the PR to use ts-jest even if it's a bit slower, if one day it becomes a problem it can be revisited and optimised.

Reply:
I've changed the PR to use ts-jest, and updated the tsconfig to turn on esModuleInterop / resolveJsonModule - which are sane defaults to have on anyways for better interoperability.

Reply:
thanks! that's everything from me. (ping @pro-wh for an approving ✅)